src/styles/pages/manage-events.module.scss

```diff
@@ -1,19 +1,105 @@
+@use 'src/styles/vars.scss' as vars;
+
 .container {
   display: flex;
   flex-direction: column;
   gap: 1rem;
+  margin: 0 auto;
+  max-width: 60rem;
 
   .header {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+
+    .viewControls {
+      .active {
+        cursor: pointer;
+        fill: var(--theme-primary-4);
+      }
+
+      .inactive {
+        cursor: pointer;
+        fill: var(--theme-primary-1);
+      }
+    }
+  }
+
+  .controls {
+    display: flex;
+    justify-content: space-between;
+
+    .controlsLeft {
+      display: flex;
+      flex-direction: row;
+      gap: 1rem;
```

Comment:
It doesn't look so great on mobile. I feel like there might be some events team people who'll be editing their event while on the go, so I think it's important that it's at least usable on mobile. Adding `flex-wrap: wrap;` to `.eventInfo`, `.controls` (as well as `gap: 1rem`), and its two children should help:

Current | With `flex-wrap`
--- | ---
![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/db732500-4ce2-4c08-8d84-975a36105c7c) | ![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/570f49fa-0304-4d00-ba61-baef16b109dc)